LARI/LARI.csproj

```
@@ -46,10 +46,10 @@
       <RequiredTargetFramework>4.0</RequiredTargetFramework>
     </Reference>
     <Reference Include="UWLARI">
-      <HintPath>..\..\..\AFSL\UWSDK\UWSDK\UWLARI\obj\Debug\UWLARI.dll</HintPath>
+      <HintPath>..\..\Users\16pat\Documents\Perforce\UWSDK\UWSDK\UWLARI\bin\Debug\UWLARI.dll</HintPath>
     </Reference>
     <Reference Include="UWWPF">
-      <HintPath>..\..\..\AFSL\UWSDK\UWSDK\UWWPF\obj\Debug\UWWPF.dll</HintPath>
+      <HintPath>..\..\Users\16pat\Documents\Perforce\UWSDK\UWSDK\UWWPF\bin\Debug\UWWPF.dll</HintPath>

```

Should not change this part of the file
```suggestion
      <HintPath>..\..\..\AFSL\UWSDK\UWSDK\UWWPF\obj\Debug\UWWPF.dll</HintPath>
```